assets/src/blocks/Accordion/AccordionEditor.js

```diff
@@ -0,0 +1,314 @@
+          accordion_btn_url: attributes.btn_url_2
+        };
+        accordion_rows.push(Object.assign({}, accordion_row));
+      }
+
+      if (attributes.accordion_headline.headline_3) {
+        let accordion_row = {
+          accordion_id: attributes.id_3,
+          accordion_headline: attributes.headline_3,
+          accordion_text: attributes.text_3 || '',
+          accordion_btn_text: attributes.btn_text_3,
+          accordion_btn_url: attributes.btn_url_3
+        };
+        accordion_rows.push(Object.assign({}, accordion_row));
+      }
+
+      if (attributes.accordion_headline.headline_4) {
+        let accordion_row = {
+          accordion_id: attributes.id_4,
+          accordion_headline: attributes.headline_4,
+          accordion_text: attributes.text_4 || '',
+          accordion_btn_text: attributes.btn_text_4,
+          accordion_btn_url: attributes.btn_url_4
+        };
+        accordion_rows.push(Object.assign({}, accordion_row));
+
+
+      }
+
+      if (attributes.accordion_headline.headline_5) {
+        let accordion_row = {
+          accordion_id: attributes.id_5,
+          accordion_headline: attributes.headline_5,
+          accordion_text: attributes.text_5 || '',
+          accordion_btn_text: attributes.btn_text_5,
+          accordion_btn_url: attributes.btn_url_5
+        };
+        accordion_rows.push(Object.assign({}, accordion_row));
+
+      }
+
+      return accordion_rows;
+    }
+
+    */
+
+
+  //renders the settings
+  renderEdit() {
+    const { attributes } = this.props;
+
+
+    // let { accordion_rows } = attributes;
+
+    // accordion_rows = this.getAccordionRows(accordion_rows);
+
+    return (
+      <Fragment>
+        <InspectorControls>
+          <PanelBody title={__('Settings', 'planet4-blocks-backend')}>
+            <div>
+              <p> Accordion Block. </p>
+            </div>
+          </PanelBody>
+        </InspectorControls>
+
+        {/*
+        {[accordion_rows].map((item, index) => {
+          return (
+            <div key={index}>
+               { !item.accordion_headline && !item.accordion_text &&
+                <div><b>{__('Please provide a title or a description for this accordion row to be displayed.', 'planet4-blocks-backend')}</b></div>
+              }
+
+            </div>
+          );
+        })}
+      */}
+      </Fragment>
+    );
+  }
+
+
+//renders the editor view
+  renderView() {
+    const { attributes, className } = this.props;
+    console.log(  "Attributes are: " +  this.props.name );
+
+    let { accordion_rows } = attributes;
+    // accordion_rows = this.getAccordionRows(accordion_rows);
+
+    return <Fragment>
+        <section>
+          <div className="block accordion-block my-3">
+            <header>
+              <RichText
+                tagName="h2"
+                className="page-section-header mt-3"
+                placeholder={__('Enter title for the block or leave empty to hide it.', 'planet4-blocks-backend')}
```

Comment:
Other WYSIWYG blocks just use `Enter title` as placeholder (for example [Articles block](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/blob/master/assets/src/blocks/Articles/ArticlesEditor.js#L112)) so we should probably use the same here, for consistency? Also, character limit is usually less for titles, from a quick search in our existing blocks I've seen either 40 or 60 (this should probably be discussed with design actually, ideally all blocks should have the same value!)